docs/VBO editor documents/components.md

```diff
@@ -1,4 +1,8 @@
-# VBO Developer Docs
+# Ontology components
+
+An ontology can include information external to the actual ontology, such as "imports" and "components" (more details [here](https://obofoundry.org/COB/odk-workflows/RepositoryFileStructure/#Components)). **Imports** represent subsets of external ontologies that are re-used in the ontology (e.g. COB, RO, OMO are ontologies imported into VBO). **Components** are part of the ontology but are managed/maintained outside of the ontology; for example, a component is a part of the ontology managed in ROBOT templates.
+
```

Comment:
@matentzn I don't know that it is the correct way to explain what a "component" is. Could you please take a look at this? I want to make sure it is not "completely wrong" before I publish this. Thank you!!

Reply:
This should probably be explained as part of the general ODK documentation, not VBO specific, but ok! Your definition looks right!

do not ever use spaces in folder or file names.

Reply:
> This should probably be explained as part of the general ODK documentation, not VBO specific, but ok! Your definition looks right!
I will make a suggestion to add that to the ODK documentation.

> do not ever use spaces in folder or file names.
Thank you for catching this!